backend/tracim_backend/models/context_models.py

```diff
@@ -539,6 +540,7 @@ def __init__(
         self,
         label: str,
         description: str,
+        access_type: str = WorkspaceAccessType.CONFIDENTIAL.value,
```

Comment:
This is funny to take a str argument (converted from `WorkspaceAccessType` for the default value) then build a `WorkspaceAccessType` in the `__init__`.

Reply:
yes, i don't know how to deal better with enum case where we:
- prefer using enum anywhere possible
- get string from the api itself.

Maybe with some marshmallow code ?
The problem is also that we do use custom validator which validate string and not "enum", which is nicely supported by hapic giving a list of valid value in autogenerated doc.